share/cluster/grpc.go

```diff
@@ -94,6 +94,8 @@ func NewGRPCServerTCP(endpoint string) (*GRPCServer, error) {
 		return nil, fmt.Errorf("failed to create new server credentials: %w", err)
 	}
 
+	//to avoid go-linter "SA1019: grpc.NewGZIPDecompressor is deprecated: use package encoding/gzip. (staticcheck)"
```

Comment:
Can we add `TODO: addressing go-linter....` here? Same below, so we can address it later and not forget about it.

Reply:
I meant...
`TODO: addressing go-linter "SA1019: grpc.NewGZIPDecompressor is deprecated: use package encoding/gzip. (staticcheck)"`